tests/net/context/src/main.c

```diff
@@ -998,43 +1034,48 @@ static void test_init(void)
 
 	/* The semaphore is there to wait the data to be received. */
 	k_sem_init(&wait_data, 0, UINT_MAX);
+
+	return NULL;
 }
 
-/*test case main entry*/
-void test_main(void)
+static void net_ctx_common_tests(void)
```

Comment:
Could we rename this to `net_ctx_common_order_dependent_tests`? I want it to be clear why we're running them all together in this particular way under a single function instead of making each test a ZTEST.

Reply:
OK